### PR TITLE
♻️Rename reaction components to interactive components

### PR DESCRIPTION
**Pull request by mszylkowski**
Reaction components are now interactive components.

### CODE REVIEW

**Comment by mszylkowski** [comment]
> Wdyt about using either interactive or interaction, but not mix both? It's clear in our minds right now but will get confusing after a few months

In that case, changed everything to "interactive" since there's already a lot of things named "interaction"

**Comment by mszylkowski** [comment]
@jridgewell missing owners approval for build-system/... files, can you take a look at them?

File: examples/visual-tests/amp-story/amp-story-reaction-quiz-sizing-positioning.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html ⚡>
```
Comment (gmajoulet) [inline examples/visual-tests/amp-story/amp-story-reaction-quiz-sizing-positioning.html:2]: Plz rename to `-interactive-`

Comment (gmajoulet) [reply examples/visual-tests/amp-story/amp-story-reaction-quiz-sizing-positioning.html:2]: `amp-story-interactive-quiz-sizing-positioning.html`

**Pull request closed without merging**